politeiawww/api/cms/v1/v1.go

```diff
@@ -123,22 +130,27 @@ type InvoiceDetailsReply struct {
 // InvoiceInput is the expected structure of the invoice.json file being added to InvoiceRecords.
 // Users' raw csv will be inputted and parsed to help in their creation.
 type InvoiceInput struct {
-	ID        string           `json:"id"`    // Optional field for contractor ID entry
-	Month     uint16           `json:"month"` // Month of Invoice
-	Year      uint16           `json:"year"`  // Year of Invoice
-	LineItems []LineItemsInput `json:"lineitems"`
+	Version            uint             `json:"version"`            // Version of the invoice input
+	Month              uint             `json:"month"`              // Month of Invoice
+	Year               uint             `json:"year"`               // Year of Invoice
+	ContractorName     string           `json:"contractorname"`     // IRL name of contractor
+	ContractorLocation string           `json:"contractorlocation"` // IRL location of contractor
+	ContractorContact  string           `json:"contractorcontact"`  // Contractor email or other contact
+	ContractorRate     uint             `json:"contractorrate"`     // Contractor Pay Rate in USD cents
+	PaymentAddress     string           `json:"paymentaddress"`     //  DCR payment address
+	LineItems          []LineItemsInput `json:"lineitems"`
 }
 
 // LineItemsInput is the expected struct of line items contained within an users'
 // invoice input.
 type LineItemsInput struct {
-	LineNumber    uint16        `json:"linenum"`       // Line number of the line item
+	LineNumber    uint          `json:"linenum"`       // Line number of the line item
```

Comment:
I actually think we can drop LineNumber altogether. Is there a reason not to use the array index as the implicit line number?